packages/ubuntu_wsl_setup/lib/app.dart

```diff
@@ -41,7 +43,7 @@ class UbuntuWslSetupApp extends StatelessWidget {
   }
 
   Widget buildWizard(BuildContext context) {
-    if (variant == null) {
+    if (variant == null && withSlides == false) {
```

Comment:
do we need a separate `withSlides` or would the existing `initialRoute` already carry this information?